vhost/src/vhost_user/gpu_backend_req.rs

```diff
@@ -156,6 +159,18 @@ impl GpuBackend {
             .map_err(|e| io::Error::new(io::ErrorKind::Other, format!("{}", e)))
     }
 
+    /// Forward 2d gpu command SetScanout requests to the frontend.
+    pub fn set_dma_scanout(&mut self, scanout: &VhostUserGpuDMABUFScanout) -> io::Result<()> {
+        self.send_message_no_reply(GpuBackendReq::DMABUF_SCANOUT, scanout, None)
+            .map_err(|e| io::Error::new(io::ErrorKind::Other, format!("{}", e)))
+    }
+
+    /// Forward 2d gpu command Update Scanout requests to the frontend.
+    pub fn get_update(&mut self, update: &VhostUserGpuUpdate) -> io::Result<()> {
+        self.send_message_no_reply(GpuBackendReq::UPDATE, update, None)
```

Comment:
You aren't passing any file descriptor here, just `None`.

Reply:
I don't think this requires a fd value, probably you mean in `set_dma_scanout` fn?

Reply:
Yes, I meant that one.